src/it/test-projects/ecocode-java-plugin-test-project/tool_send_to_sonar.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env sh
+
+# "sonar.token" variable (or sonar.login before SONARQUBE 9.9) : private TOKEN generated in your local SonarQube during installation
+# (input paramater of this script)
+mvn org.sonarsource.scanner.maven:sonar-maven-plugin:3.9.1.2184:sonar -Dsonar.host.url=http://localhost:$1 -Dsonar.token=$2
```

Comment:
check the current repository to get the port variable

Reply:
This `tool_send_to_sonar.sh` file comes from [`ecoCode-java-test-project` project](https://github.com/green-code-initiative/ecoCode-java-test-project/blob/main/tool_send_to_sonar.sh).

It is no longer necessary in this context.

I will remove this file.